recOrder/plugin/main_widget.py

```diff
@@ -1154,6 +1154,7 @@ def _add_or_update_image_layer(
         move_to_top : bool, optional
             whether to move the updated layer to the top of layers list, by default True
         """
+        image = np.squeeze(image)
```

Comment:
Is this supposed to squeeze out the Z dimension?

Reply:
That's right, and I think it's most intuitive to squeeze everything. Happy to chat.